definitions/features/cron.rb

```diff
@@ -0,0 +1,25 @@
+class Features::Cron < ForemanMaintain::Feature
+  metadata do
+    label :cron
+    confine do
+      ForemanMaintain.config.manage_crond && !(
+        feature(:downstream) && feature(:downstream).less_than_version?('6.3')
+      )
+    end
+  end
+
+  def services
+    # TODO: For debian, add cron as service
+    [
+      system_service('crond', 5)
```

Comment:
Hmm.. Thank you @mbacovsky  for pointing this.

I don't think it will bring any harm for other services but the only thing I could see here is that whenever command  `foreman-maintain service restart` will run, it won't run jobs scheduled at that moment.
Again, this needs to be confirmed.

Reply:
@kgaikwad I think this still needs some resolution. I believe the general opinion was not to touch the crond. We made it opt-in for maintenance mode but `crond` is also stopped during `formena-maintain service stop`. Do we want to keep this behavior? If so we should update the comments in the configs and warn the users.

Reply:
@mbacovsky, 
Thank you for review. Yes, right!
With these changes,  `crond` will get stop when `foreman-miantain service stop` when user set `manage_crond:true`. I will update PR which will fix this behaviour.